Practical Example/IosInterviewPracticeProject/IosInterviewPracticeProject/Constants/ContentType.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ContentType.swift
+//  IosInterviewPracticeProject
+//
+//  Created by Robert Ramirez on 10/20/20.
+//  Copyright © 2020 me.robert.ramirez. All rights reserved.
+//
+
+import Foundation
+
+enum ContentType {
+  static let article = "article"
+  static let collection = "collection"
+}
```

Comment:
Another way to write this would be

```swift
enum ContentType: String {
    case article = "article"
    case collection = "collection"
  }
```

This would have a few benefits line being able to switch on it more nicely, init it with the raw value, add Codable support etc.